server/go/server.go

```diff
@@ -24,6 +24,7 @@ func main() {
 	stripe.Key = os.Getenv("STRIPE_SECRET_KEY")
 
 	http.Handle("/", http.FileServer(http.Dir(os.Getenv("STATIC_DIR"))))
+	http.HandleFunc("/checkout-session", handleGetCheckoutSession)
```

Comment:
to be consistent with the other handlers, I would replace `handleGetCheckoutSession` with `handleCheckoutSession`